dt-backend/src/model/user/user.service.ts

```diff
@@ -35,8 +36,7 @@ export class UserService {
     })
   }
 
-  async updateUser(params: {where: Prisma.UserWhereUniqueInput; data: Prisma.UserUpdateInput;}): Promise<User> {
-    const { data, where } = params;
+  async updateUser(data: UserUpdateDto["data"], where: UserUpdateDto["where"]): Promise<User> {
```

Comment:
Yeah, replied on the last review, that this is not exactly how it should be done ^^. Check it there 😉. It's just a minor change, so all good )). And here you got the Dto naming right ))!